index.html

```diff
@@ -125,11 +127,7 @@ <h4 class="header-1">Multi-Post Stories Gain+Glory</h4>
                 <h1 class="header-2">About me</h1>
                
 
-                <p class="paragraph"> I can help you build a product , feature or website Look through some of my work
-                    and
-                    experience! If you like
-                    what you see and have a project you need coded,
-                    don’t hesitate to contact me.
+                <p class="paragraph-1"> I can help you build a product , feature or website Look through some of my work and <br>experience! If you like what you see and have a project you need coded, <br>don’t hesitate to contact me.
```

Comment:
@joskalenda I don't know why it is behaving that way on your machine since I have arranged the items as they should and from my side, it doesn't have the scrolling effect to the right

Reply:
![Screenshot from 2023-05-12 08-42-16](https://github.com/NgachaIan/PersonalPortfolio/assets/9462888/c57d8d85-f958-4ff6-bfa3-e0ba9cbcfcbb)
@joskalenda this is how it's supposed to look like

Reply:
Hey @NgachaIan , I see. Would you please make a small video showing that? and please put your screen size to 768px and let's start investigating from that, may be you have changes in your local that you did not push yet

Reply:
@joskalenda okay, will do though the changes in my local reflect in this branch, kindly check again

Reply:
https://github.com/NgachaIan/PersonalPortfolio/assets/9462888/af1f94d8-81fc-45a3-82ca-21e0e89b24e3

@joskalenda  here is the screen recording